examples/with-realm-web/pages/index.js

```diff
@@ -0,0 +1,84 @@
+import useSWR from 'swr'
+import { generateAuthHeader, REALM_GRAPHQL_ENDPOINT } from '../lib/RealmClient'
+import { FIND_MOVIES } from '../lib/graphql-operations'
+
+const fetcher = async (url) =>
+  fetch(url, {
+    method: 'POST',
+    headers: await generateAuthHeader(),
+    body: JSON.stringify({
+      query: FIND_MOVIES,
+    }),
+  }).then((res) => res.json())
+
+const IndexPage = () => {
+  const { data, error } = useSWR(REALM_GRAPHQL_ENDPOINT, fetcher)
```

Comment:
Let's change this, use the query as the variable instead of the URL, because for GraphQL the URL is mostly always the same.

It would be something like this:

```js
const fetcher = async (query) =>
  fetch(REALM_GRAPHQL_ENDPOINT, {
    method: 'POST',
    headers: await generateAuthHeader(),
    body: JSON.stringify({ query }),
  }).then((res) => res.json())

const { data, error } = useSWR(FIND_MOVIES, fetcher)
```